ui/app/styles/utils/_size_variables.scss

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+/* General sizing in rem values used largely for text sizing.*/
+$size-1: 3rem; // 48px, same as $spacing-xxl
+$size-2: 2.5rem; // 40px
+$size-3: (24/14) + 0rem; // ~1.714rem ~27px
+$size-4: 1.5rem; // 24px same as $spacing-l
+$size-5: 1.25rem; // 20px
+$size-6: 1rem; // 16px same as $spacing-m
+$size-7: (13/14) + 0rem; // ~.929rem ~15px
+$size-8: (12/14) + 0rem; // ~.857rem  ~13.7px
+$size-9: 0.75rem; // 12px same as $spacing-s
+$size-10: 0.5rem; // 8px same as $spacing-xs
+$size-11: 0.25rem; // 4px same as spacing-xxs
+
+/* Spacing vars in px */
+$spacing-xxs: 4px;
+$spacing-xs: 8px;
+$spacing-s: 12px;
+$spacing-m: 16px;
+$spacing-l: 24px;
+$spacing-xl: 36px;
+$spacing-xxl: 48px;
+
+/* Border radius */
+$radius: 2px;
+$radius-large: 4px;
+
+/* Responsiveness */
+// 960, 1152, and 1344 have been chosen because they are divisible by both 12 and 16
+$mobile: 769px;
+// 960px container + 4rem
+$desktop: 960px + (2 * 32px);
+// 1344px container + 4rem
+$fullhd: 1344px + (2 * 32px);
+
+/* Animations and transitions */
+$speed: 150ms;
+$speed-slow: $speed * 2;
+$easing: ease-out;
```

Comment:
Should we name this this var to be `$ease-out`? No need to change this since it already went through PR review - just a thought! :)

Reply:
Yeah, I agree the name isn't great. It's a carry over from bulma so I'm just keeping as is for now.